model_analyzer/config/input/config_field.py

```diff
@@ -200,3 +207,9 @@ def required(self):
 
     def set_name(self, name):
         self._field_type.set_name(name)
+
+    def set_by_config(self) -> bool:
```

Comment:
any function that starts with `set` implies that it is taking an input and setting something. Can you change this to `is_set_by_config` or `is_specified` or something else along those lines?